src/gui_common/CellHexPhotoBuilder.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using Godot;
+
+public class CellHexPhotoBuilder : Spatial, IPhotographable
```

Comment:
I'd move this class into the microbe stage folder as photographing the hexes of a cell are not a common operation, instead it sounds like a microbe specific thing.